### ISSUE TITLE
have any plan to bulid a docker image?

### ISSUE THREAD

**Issue by luomoxu** [nadoo/glider#65]
have any plan to bulid a docker image?

**Comment by djarbz** [comment]
You can easily build your own.
Create a folder for your project and save Dockerfile and docker-compose.yml
Create a glider.conf with your configuration.
Create a rules.d folder for any rulesets.
This example listens on TCP/UDP 110 which I have found to be open outbound on many corporate networks.

**Dockerfile:**
```
FROM golang:1.11-alpine3.8

RUN apk add --no-cache git

RUN go get -u github.com/nadoo/glider

RUN pwd && ls -lah

WORKDIR /go/bin

CMD ["glider", "-config", "glider.conf"]
```

**Docker Compose File:**
```
version: '3.4' #Requires Docker version 17.12.0

services:
  glider:
    build:
      context: .
    container_name: glider
    restart: unless-stopped
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - ./glider.conf:/go/bin/glider.conf:rw
      - ./rules.d:/go/bin/rules.d:rw
    ports:
      - "110:110/tcp"
      - "110:110/udp"
```

**Comment by nadoo** [comment]
Someone did it:
https://hub.docker.com/r/kometchtech/glider

**Comment by rustycl0ck** [comment]
But that is not up to date. Currently, the latest release here is `0.12.2` while the docker image is available only for `0.10.4`.
